src/components/Footer/Footer.jsx

```diff
@@ -1,8 +1,144 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import './Footer.scss';
 
 function Footer() {
-  return <h1>footer</h1>;
+  return (
+    <div className="footerWrapper">
```

Comment:
```suggestion
    <div className="footer">
```
가장 최상위 부모태그의 클래스네임은 컴포넌트의 이름과 동일하게 해주는 것이 위코드 컨벤션입니다!

Reply:
수정했습니다!